jsdependency/jspackage.go

```diff
@@ -73,7 +96,7 @@ func toolCommandBuilder(packageManger Tool, command managerCommand) string {
 	return "add"
 }
 
-func runManagerCmd(packageManager Tool, packageManagerCmd string, commandScope CommandScope, pkg ...string) error {
+func runManagerCmd(packageManager Tool, packageManagerCmd string, commandScope CommandScope, pkg ...string) (*command.Model, error) {
```

Comment:
please rename this function as it does not run the commands anymore, something like: `createManagerCmd / managerCmd /...`